src/incidents/view/ViewIncidentDetails.tsx

```diff
@@ -132,6 +132,17 @@ function ViewIncidentDetails(props: Props) {
           />
         </Column>
       </Row>
+      {data.patient && (
+        <Row>
+          <Column md={6}>
+            <TextInputWithLabelFormGroup
+              label={t('incidents.reports.patient')}
+              name="patient"
+              value={data.patient}
```

Comment:
We want to display the patient name here instead of the id

![image](https://user-images.githubusercontent.com/5468546/103351133-e0b84300-4aaa-11eb-8930-3083ab2589d3.png)

See the Appointment view for an example

![image](https://user-images.githubusercontent.com/5468546/103351164-fcbbe480-4aaa-11eb-9db4-9de5696731de.png)